sdk/eventgrid/ci.data.yml

```diff
@@ -0,0 +1,43 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+# This is for Track 1 data plane SDK
+
+trigger:
+  branches:
+    include:
+    - main
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/eventgrid/ci.data.yml
+    - sdk/eventgrid/Microsoft.Azure.EventGrid/
+    - sdk/eventgrid/Microsoft.Azure.Messaging.EventGrid.CloudNativeCloudEvents/
+    - sdk/eventgrid/Microsoft.Azure.WebJobs.Extensions.EventGrid/
+
+pr:
+  branches:
+    include:
+    - main
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/eventgrid/ci.data.yml
+    - sdk/eventgrid/Microsoft.Azure.EventGrid/
+    - sdk/eventgrid/Microsoft.Azure.Messaging.EventGrid.CloudNativeCloudEvents/
```

Comment:
This one is actually maintained by our team.

Reply:
moved to ci.yml